Source/Shared/Paths.cs

```diff
@@ -3,36 +3,43 @@
 
 namespace CodeImp.Bloodmasters;
 
-public static class Paths
+public class Paths
 {
     private const string DevModeMarkerFileName = ".bloodmasters.dev.marker";
     private const string DevSolutionRootMarkerFileName = ".bloodmasters.solution-root.marker";
     private const string TargetFrameworkForExecutables = "net7.0-windows";
-    private const string BuildConfiguration =
+    private readonly string _buildConfiguration =
 #if DEBUG
         "Debug";
 #else
         "Release";
 #endif
 
+    public Paths(bool isDev = false)
+    {
+        if (isDev)
+            _buildConfiguration = "Debug";
+    }
```

Comment:
This flag is supposed to override the `IsDevModeBuild`, not the `_buildConfiguration`.